{{ cookiecutter.package_name }}/{{ cookiecutter.module_name }}/conftest.py

```diff
@@ -3,47 +3,37 @@
 import os
 
 from astropy.version import version as astropy_version
+
+# For Astropy 3.0 and later, we can use the standalone pytest plugin
 if astropy_version < '3.0':
-    # With older versions of Astropy, we actually need to import the pytest
-    # plugins themselves in order to make them discoverable by pytest.
-    from astropy.tests.pytest_plugins import *
+    from astropy.tests.pytest_plugins import *  # noqa
+    del pytest_report_header
 else:
-    # As of Astropy 3.0, the pytest plugins provided by Astropy are
-    # automatically made available when Astropy is installed. This means it's
-    # not necessary to import them here, but we still need to import global
-    # variables that are used for configuration.
-    from astropy.tests.plugins.display import (pytest_report_header,
-                                               PYTEST_HEADER_MODULES,
-                                               TESTED_VERSIONS)
-
-from astropy.tests.helper import enable_deprecations_as_exceptions
-
-## Uncomment the following line to treat all DeprecationWarnings as
-## exceptions. For Astropy v2.0 or later, there are 2 additional keywords,
-## as follow (although default should work for most cases).
-## To ignore some packages that produce deprecation warnings on import
-## (in addition to 'compiler', 'scipy', 'pygments', 'ipykernel', and
-## 'setuptools'), add:
-##     modules_to_ignore_on_import=['module_1', 'module_2']
-## To ignore some specific deprecation warning messages for Python version
-## MAJOR.MINOR or later, add:
-##     warnings_to_ignore_by_pyver={(MAJOR, MINOR): ['Message to ignore']}
-# enable_deprecations_as_exceptions()
+    from pytest_astropy_header.display import PYTEST_HEADER_MODULES, TESTED_VERSIONS
+
+
+def pytest_configure(config):
```

Comment:
Not sure this is working properly - I'm seeing `Pandas` listed as not available, instead of simply omitted.

Reply:
Hmm, since `astropy_helpers` is still listed, there might be some out-of-date cruft getting in the way.

Reply:
Weird. I tried just adding a print statement inside `conftest.pytest_configure` and it only gets shown inside `collecting...` - indeed, if I just add a breakpoint inside that function, the break also happens after the header is printed. 

If I instead add a new `conftest.py` in my root directory, the removals from `PYTEST_HEADER_MODULES` do work (though for some reason it always list astropy_helpers).

So, here, it may be most relevant to have a `conftest.py` in the root directory; the rest may be more of an issue for `pytest_astropy_header`

Reply:
And for `pytest_astropy_header`, the present of `astropy-helpers` is of course already addressed in https://github.com/astropy/pytest-astropy-header/pull/16

Reply:
OK, the following `conftest.py` in the root directory works at least for me:
```
import os

try:
    from pytest_astropy_header.display import PYTEST_HEADER_MODULES, TESTED_VERSIONS
except ImportError:
    PYTEST_HEADER_MODULES = {}
    TESTED_VERSIONS = {}


def pytest_configure(config):

    config.option.astropy_header = True

    # Customize the following lines to add/remove entries from the list of
    # packages for which version numbers are displayed when running the tests.
    PYTEST_HEADER_MODULES.pop('Pandas', None)
    PYTEST_HEADER_MODULES.pop('h5py', None)
    PYTEST_HEADER_MODULES.pop('Matplotlib', None)
    PYTEST_HEADER_MODULES.pop('Scipy', None)
    PYTEST_HEADER_MODULES.pop('astropy-helpers', None)
    PYTEST_HEADER_MODULES['Astropy'] = 'astropy'

    from baseband.version import version
    packagename = os.path.basename(os.path.dirname(__file__))

    TESTED_VERSIONS[packagename] = version

# Uncomment the last two lines in this block to treat all DeprecationWarnings as
# exceptions. For Astropy v2.0 or later, there are 2 additional keywords,
# as follow (although default should work for most cases).
# To ignore some packages that produce deprecation warnings on import
# (in addition to 'compiler', 'scipy', 'pygments', 'ipykernel', and
# 'setuptools'), add:
#     modules_to_ignore_on_import=['module_1', 'module_2']
# To ignore some specific deprecation warning messages for Python version
# MAJOR.MINOR or later, add:
#     warnings_to_ignore_by_pyver={(MAJOR, MINOR): ['Message to ignore']}
from astropy.tests.helper import enable_deprecations_as_exceptions  # noqa
enable_deprecations_as_exceptions()
```

Reply:
would the `    config.option.astropy_header = True` work if you ended up in the except part of the top try/except? In a similar situation I simply put the whole stuff in the try part.

Reply:
If you already have `baseband` in this line: `from baseband.version import version`, then why do the next two as a hackary for packagename, etc?

Reply:
Ah, yes, that was rather silly! It does seem that ideally one would just have this stuff in `setup.cfg`.

Reply:
I'll comment in the main thread since this is a big-ish issue